earth2studio/models/px/sfno.py

```diff
@@ -146,9 +148,11 @@ def __init__(
         self.model = core_model
         self.register_buffer("center", center)
         self.register_buffer("scale", scale)
+        self.VARIABLES = self.load_variables()
```

Comment:
Make the variables explicit in the model initialization, then set the default to be the global `VARIABLES` as above.